packages/core/contracts/common/implementation/Withdrawable.sol

```diff
@@ -23,7 +23,7 @@ abstract contract Withdrawable is MultiRole {
      * @notice Withdraws ETH from the contract.
      */
     function withdraw(uint256 amount) external onlyRoleHolder(roleId) {
-        Address.sendValue(msg.sender, amount);
+        Address.sendValue(payable(address(msg.sender)), amount);
```

Comment:
tiny nit: according to [this](https://docs.soliditylang.org/en/v0.8.4/080-breaking-changes.html#new-restrictions), you should be able to just do `payable(msg.sender)` rather than `payable(address(msg.sender))`.